eav/models.py

```diff
@@ -406,9 +406,11 @@ def _set_value(self, new_value):
 
     value = property(_get_value, _set_value)
 
-    def __unicode__(self):
-        return u"%s - %s: \"%s\"" % (self.entity, self.attribute.name,
-                                     self.value)
+    def __str__(self):
+        return '{}: "{}" ({})'.format(self.attribute.name, self.value, self.entity)
+
+    def __repr__(self):
+        return '{}: "{}" ({})'.format(self.attribute.name, self.value, self.entity.pk)
```

Comment:
@IwoHerka is there a particular reason for different `__str__` and `__repr__` with regards to last tuple element (model vs pk)?

Reply:
Yes, it a convenience. `entity` is the owner a value. Therefore, in `__str__` we want to display this object for the end user. However, for debugging (`__repr__`), it is more convenient to identify owners simply via their `pk`s.

Reply:
👍